app/generators/questionnaire_generator.rb

```diff
@@ -658,6 +662,36 @@ def generate_raw(question)
       question[:content].html_safe
     end
 
+    def generate_unsubscribe(question)
+      body = safe_join([
+                         generate_unsubscribe_content(question),
+                         generate_unsubscribe_action(question)
+                       ])
+      body = content_tag(:div, body, class: 'card light-grey-background-color')
+      body
+    end
+
+    def generate_unsubscribe_content(question)
+      body = []
+      body << content_tag(:span, question[:title].html_safe, class: 'card-title') if question[:title].present?
+      body << content_tag(:p, question[:content].html_safe) if question[:content].present?
+      body = safe_join(body)
+      body = content_tag(:div, body, class: 'card-content black-text')
+      body
+    end
+
+    def generate_unsubscribe_action(question)
+      response = Response.find_by_id(question[:response_id])
+      url_href = '#'
```

Comment:
willen we dat het # blijft als er geen response is?

Reply:
Ja daarom had ik het zo gemaakt. Het enige geval waar het voorkomt dat we een vragenlijst renderen zonder dat er een response is is volgens mij bij het previewen van de questionnaire in de admin interface. Maar het komt dus wel voor, en dan moet niet alles kapotgaan.